viewmaster/link.py

```
@@ -22,36 +37,95 @@
 #     LOG_ROOT_PREFIX_ = '/Library/WebServer/Logs/webapps/'
 ################################################################################
 
-from viewmaster_config import *
+from .viewmaster_config import *
```

Ruff (0.14.10)

`from .viewmaster_config import *` used; unable to detect undefined names

(F403)